site/sfguides/src/intro_to_machine_learning_with_snowpark_ml_for_python/intro_to_machine_learning_with_snowpark_ml_for_python.md

```diff
@@ -144,19 +151,30 @@ Open the following notebook in Snowflake Notebooks and run each of the cells: [2
 
 In this notebook, we will illustrate how to train an XGBoost model with the `diamonds` dataset using the Snowpark ML Modeling. We also show how to execute batch inference and model explainability through the Snowflake Model Registry.
 
+<!-- ------------------------ -->
+## Advanced MLOps: Managing ML Models from Iteration to Production
+Duration: 15
+
+Open the following notebook in Snowflake Notebooks and run each of the cells: [3_sf_nb_snowpark_ml_adv_mlops.ipynb](https://github.com/Snowflake-Labs/sfguide-intro-to-machine-learning-with-snowflake-ml-for-python/blob/main/notebooks/3_sf_nb_snowpark_ml_adv_mlops.ipynb)
+
+In this notebook, we will show you how to manage Machine Learning models from experimentation to production using existing (Snowpark ML Modeling & Model Registry) and new Snowflake MLOps features:
+- [Snowflake Datasets](https://docs.snowflake.com/en/developer-guide/snowflake-ml/dataset) for creating and managing training data
+- [Snowflake ML Lineage](https://docs.snowflake.com/en/developer-guide/snowflake-ml/ml-lineage) to trace end-to-end data and model lineage 
+
+We will also go more into detail in using the Model Registry API.
+
 <!-- ------------------------ -->
 ## Conclusion
```

Comment:
@sfc-gh-sidas Please rename this to "Conclusion And Resources"

Reply:
Done!